kalite/control_panel/templates/control_panel/partials/_groups_table.html

```diff
@@ -70,9 +70,9 @@
                                         {% endif %}
                                     </td>
                                     <td>
-                                        {% if group.id %}
+
                                         {# Translators: this is a verb; by clicking this link, the user will be able to coach students. #}
-                                        <a title="{% blocktrans with groupname=group.name %}Coach group {{ groupname }}.{% endblocktrans %}" href="{% url 'coach_reports' %}{{ facility_id }}/{{ group.id }}/">
+                                        <a title="{% blocktrans with groupname=group.name %}Coach group {{ groupname }}.{% endblocktrans %}" href="{% url 'coach_reports' zone_id=zone_id %}{{ facility_id }}/{% if group.id %}{{ group.id }}{% else %}{{ ungrouped_id }}{% endif %}/">
```

Comment:
I see zone_id getting passed into the coach reports context, but does this table get used elsewhere in the control panel? Or did it already have zone_id?